kairon/shared/data/processor.py

```diff
@@ -3539,6 +3539,41 @@ def add_default_fallback_data(self, bot: Text, user: Text, nlu_fallback: bool =
             if not Utility.is_exist(Responses, raise_error=False, bot=bot, status=True, name__iexact=DEFAULT_NLU_FALLBACK_UTTERANCE_NAME):
                 self.add_text_response(DEFAULT_ACTION_FALLBACK_RESPONSE, DEFAULT_NLU_FALLBACK_UTTERANCE_NAME, bot, user)
 
+    def add_default_training_data(self, bot: Text, user: Text, default_data: bool = True):
+        if default_data:
+            data = Utility.read_yaml("template/use-cases/GPT-FAQ/data/nlu.yml")
+            utterance = Utility.read_yaml("template/use-cases/GPT-FAQ/domain.yml")
+            for name in data['nlu']:
+                self.add_intent(name['intent'], bot, user, is_integration=False)
+                examples = name['examples'].split('\n')
+                examples = [exmp.replace('- ', '') for exmp in examples if exmp]
+                self.add_training_example(examples, name['intent'], bot, user, is_integration=False)
+                for key, value in utterance['responses'].items():
```

Comment:
why are you unnecessarily iterating over dictionary when you already know the key for which you want to retrieve the value. this defeats the purpose if using dict!!